python-sdk/tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -195,13 +203,28 @@ def handle_failure(context):
         return dag
 
 
+def is_skipped(dataset: dict, database_name: str) -> bool:
+    """
+    Check if the dataset is skipped for database. Default value is False.
+    :param database_name: name of db expected in database_kwargs
+    :param dataset: dataset config
+    """
+    database_kwargs = dataset["database_kwargs"]
+    if database_name not in database_kwargs:
+        return False
+    elif "skip" not in database_kwargs[database_name]:
+        return False
```

Comment:
We might not need this check. The below return bool statement would return False if the 'skip' is absent, because bool(None) is False.